src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -782,10 +779,9 @@ bool OutputStateMachineEngine::ActionOscDispatch(const wchar_t /*wch*/,
     case OscActionCodes::SetIconAndWindowTitle:
     case OscActionCodes::SetWindowIcon:
     case OscActionCodes::SetWindowTitle:
+    case OscActionCodes::DECSWT_SetWindowTitle:
     {
-        std::wstring title;
-        success = _GetOscTitle(string, title);
-        success = success && _dispatch->SetWindowTitle(title);
+        success = _dispatch->SetWindowTitle(string);

```

Comment:
All that `_GetOscTitle` was doing was assigning the `wstring_view` to a `wstring`, and checking that it wasn't blank. But we don't actually want that blank check, and there's no reason why we can't just pass the `wstring_view` through to `SetWindowTitle` as is.